content/turinge/drone-portainer/index.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2020-07-16T00:00:00+00:00
+title: Portainer Stack Deploy
+author: turinge
+tags: [ deployment]
+logo: portainer.svg
+repo: turinge/drone-portainer
+image: turinge/drone-portainer
+---
+
+The drone plugin can be used to deploy a Docker image to a Drone environment. [source](https://github.com/LGinC/drone-portainer) <br>
+The below pipeline configuration demonstrates simple usage:
+
+```yaml
+steps:
+- name: portainer
+  image: turinge/drone-portainer
+  settings:
+    serverurl: http://xxxxx:9000
+    username: 
+      from_secret: portainer_username
+    password:
+      from_secret: portainer_password
+    endpointId: 1
+    stackname: xxxservice
+    imagename: xxx/xxx
+    docker_compose: |
+      version: "2"
+      services:
+        xxx:
+          image: xxx/xxx
+          ports:
+          - 80:80
+```
+# Parameter Reference
+
+serverurl
+: portainer server url. like this: http://xxx.com:9000
+
+username
+: portainer username
+
+password
+: portainer password
+
+endpointId
+: portainer endpoint id,default 1, localhost is 1
+stackname
+: name of stack, show in stack list
+
+imagename
+: name of pull image, like: mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine
+
+docker_compose
+: content of docker-compose.yml. ps: portainer just support version: "2"  <br>
```

Comment:
`content of docker-compose, portainer supports version 2 only` should be fine, remove the rest.

Reply:
done